### PR TITLE
More complete about tab details

### PR DESCRIPTION
**Pull request by johnbradley**
Details based on document from Raluca.

### CODE REVIEW

**Comment by johnbradley** [comment]
screen shot 2016-11-07 at 7 54 33 am

screen shot 2016-11-07 at 7 54 42 am

**Comment by johnbradley** [comment]
screen shot 2016-11-07 at 8 08 26 am

screen shot 2016-11-07 at 8 08 30 am

**Comment by johnbradley** [comment]
@dleehr How do the above two screenshots look to you?

File: portal/src/app/about/AboutContent.jsx [hunk 1]
```diff
@@ -11,19 +12,99 @@ class AboutContent extends React.Component {
...
+                </p>
+            </div>
+            <p className="AboutContent_paragraph">
+                All predictions can be downloaded in tab-delimited and csv formats. All data sources used in the iMADS
+                web application are described in the DATA SOURCES tab.
+            </p>
+            <div>
+                <span className="AboutContent_group_label">TECHNOLOGY</span>
+                <ul>
+                    <li>Database management system: PostgreSQL</li>
```
Comment (hlapp) [inline portal/src/app/about/AboutContent.jsx:61]: Include versions here and for the other pieces? Also, hyperlink each one of them.

File: portal/src/app/about/AboutContent.jsx [hunk 2]
```diff
@@ -11,19 +12,99 @@ class AboutContent extends React.Component {
...
+                </ul>
+            </div>
+            <div>
+                <span className="AboutContent_group_label">SOURCE CODE</span>
                 <ul>
-                    <li>Website: {websiteSource}</li>
-                    <li>TF binding predictions: {predictionSource}</li>
-                    <li>TF preference: {preferenceSource}</li>
+                    <li>Website: <a href="https://github.com/Duke-GCB/iMADS">github.com/Duke-GCB/iMADS</a></li>
+                    <li>iMADS Worker: <a href="https://github.com/Duke-GCB/iMADS-worker">
```
Comment (hlapp) [inline portal/src/app/about/AboutContent.jsx:74]: What the _iMADS Worker_ does hasn't been mentioned yet. Use a more descriptive label, such as _Worker for prediction server_?

**Review by hlapp — COMMENTED** [review]
See minor inline comments. Also, any reason we shouldn't be using heading markup for the headings instead of `<span>`? For example, `<h4>` or `<h5>`?

**Pull request merged**